src/client/App.js

```diff
@@ -26,7 +27,10 @@ function App() {
         <Route exact path="/">
           <LandingPageContainer />
         </Route>
-
+        {/* Category page */}
+        <Route exact path="/category/:name">
```

Comment:
For now your page is not accesible from menu - so you should talk to Saru to change menu component or change this path

Reply:
done